public/app/plugins/datasource/azuremonitor/components/ConfigEditor/AzureCredentialsForm.test.tsx

```diff
@@ -15,7 +15,7 @@ const setup = (propsFunc?: (props: Props) => Props) => {
       clientId: '34509fad-c0r9-45df-9e25-f1ee34af6900',
       clientSecret: undefined,
     },
-    azureCloudOptions: [
+    legacyAzureCloudOptions: [
```

Comment:
Why is this legacy now?

Reply:
The cloud names we use have changed, we're technically converting to the new options in the backend but going forward we'll only be using the new ones.